poetry.lock

```diff
@@ -1,11 +1,12 @@
-# This file is automatically @generated by Poetry 1.8.2 and should not be changed by hand.
+# This file is automatically @generated by Poetry 2.1.1 and should not be changed by hand.
```

Comment:
Does this pose any issues for future `poetry.lock` updates by those of us still using poetry 1.x?

Reply:
It seems like dependabot used poetry 2.1, I ran `poetry lock --no-update with 1.8.4` and it should be good now